unlock-protocol.com/blog/emergency-upgrade-1-3.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Emergency Upgrade
+subTitle: Version 1-3 is now available
+authorName: Nick Mancuso
+publishDate: January 17, 2020
+description: Explaining what went wrong and why we released a new version so quickly after our last upgrade.
+image: /static/images/blog/unlocking-smart-contracts/code.jpeg
+---
+
+We recently noticed a vulnerability which if abused could have had bad consequences – but we caught it early and fixed it promptly. This post aims to explain the errors we made with our most recent smart-contract update.
+
+There were problems with this release which if targeted by an attacker had the potential to do one of the following:
+ - Steal money made from previous key sales.
+ - Break every lock (and key) using the new version - blocking new sales, preventing key owners from being recognized and preventing the lock owner from accessing any funds previously made.
+
+We fixed the second issue already, so no worries there.  But funds are at risk if your lock is impacted.  For that we need the lock owners themselves to upgrade ASAP.
+
+This only impacts locks created in the past couple weeks. If you recently created a lock, please go to the Unlock dashboard to check if you are impacted.  The dashboard will flag any impacted lock, asking you to upgrade.
```

Comment:
Let's be more precise on dates (ideally even block numbers!)

Reply:
And lets something along the lines of "if the dashboard does not show a warning  like the one below, your lock is safe". 

[I'll send a srceenshot ASAP]

Reply:
<img width="1293" alt="image" src="https://user-images.githubusercontent.com/17735/72651244-cb591000-3950-11ea-8cc0-8b08b1c1224d.png">

Reply:
Will do.  
Issue introduced: https://etherscan.io/tx/0x159252de7b715cb7e773e5eb54435cd19a8563a574cc5ecfd4f6fcff7bc7204f
Issue resolved: https://etherscan.io/tx/0xa66dc730133db01cc39a30ea7f968558e6f4e5aa1305c5d9851e0547810f71c8